### PR TITLE
Add Redwoodjs integration

### PR DESCRIPTION
**Pull request by jakubriedl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,3 +93,38 @@ export interface AppearConfig {
   }
 }
 ```
+
+### Framework specific integrations
+
+Not all services are deployed as a node applications. If it's your case you can either use one of pre-built integrations or write your own adapter
```
Comment (SomeoneWeird) [inline README.md:99]:
```suggestion
Not all services are deployed as a node applications. If that applies to you, you can either use one of pre-built integrations, or write your own adapter.
```

File: README.md [hunk 2]
```diff
@@ -93,3 +93,38 @@ export interface AppearConfig {
...
+
+#### RedwoodJS at vercel
+
+1. Create a file, anywhere in backend side that suits in your structure, for example `api/src/withAppear.ts`
```
Comment (SomeoneWeird) [inline README.md:103]:
```suggestion
1. Create a file where you can instantiate Appear, for example: `api/src/withAppear.ts`
```

File: README.md [hunk 3]
```diff
@@ -93,3 +93,38 @@ export interface AppearConfig {
...
+  // api key you can obtain at https://app.appear.sh/settings
+  apiKey: "your-api-key",
+  // any identifier of enviroment you prefer, we recommend to at least separate production/staging/development. The more granular the better
+  environment: process.env.NODE_ENV,
+  // other config as you desire
+  // ...
+})
+```
+
+2. wrap your Serverless Functions (API Endpoints) in `withAppear`
```
Comment (SomeoneWeird) [inline README.md:119]:
```suggestion
2. Wrap your Serverless Functions (API Endpoints) in `withAppear`
```

File: README.md [hunk 4]
```diff
@@ -93,3 +93,38 @@ export interface AppearConfig {
...
+```ts
+// eg. api/src/functions/<function name>/<function name>.ts
+import { withAppear } from "src/withAppear"
+
+export const handler = withAppear(async (event, context) => {
+  // your code
+})
+```
+
+3. it's done, now you can try calls to your API and they'll appear in Appear
```
Comment (SomeoneWeird) [inline README.md:130]:
```suggestion
3. Once finished, any calls to your APIs should show up in Appear
```

File: package.json [hunk 5]
```diff
@@ -22,21 +22,23 @@
...
+    "msw": "^2.3.1",
+    "prettier": "^3.3.2",
+    "typescript": "^5.5.2",
+    "vitest": "^1.6.0"
   },
   "dependencies": {
-    "@mswjs/interceptors": "^0.26.8",
-    "browser-or-node": "3.0.0-pre.0",
+    "@mswjs/interceptors": "^0.30.0",
+    "@vercel/functions": "^1.0.2",
```
Comment (SomeoneWeird) [inline package.json:40]: Hmm. Should this be a peer dep? Pulling this in feels strange. Will using `waitUntil` just noop in a non-vercel environment? Or will it throw?

**Review by SomeoneWeird — APPROVED** [review]

File: package.json (hunk 5 above)
Comment (jakubriedl) [reply package.json:40]: making it peer dep would increase number of steps consumer have to do. And yea in non-vercel env it is noop and is working just fine. I've tested it locally and here's [the code](https://github.com/vercel/vercel/blob/main/packages/functions/index.js#L15) which doesn't do anything if `waitUntil` isn't present in `globalThis`.

**Review by SomeoneWeird — APPROVED** [review]

**Pull request merged**